frontend/src/modules/batchactions/components/BatchActions.js

```diff
@@ -80,6 +80,10 @@ export class BatchActionsBase extends React.Component<InternalProps> {
     }
 
     approveAll = () => {
+        if (this.props.batchactions.fetching) {
+            return;
+        }
```

Comment:
Maybe we should also disable the various buttons while a fetching is in progress? That would make things clearer for users. I think a button that just doesn't do anything when clicked, without any feedback, is not a great user experience.

Reply:
You mean by setting the `disabled` attribute?

Reply:
Yup. And maybe changing their color or something?

Reply:
Sure thing!